R/index_cfactor.R

```diff
@@ -1,112 +1,109 @@
 #'  that will be given.
-#' @param ... Further arguments to be passed to or from other methods, 
+#' @param ... Further arguments to be passed to or from other methods,
 #'  in particular to \code{\link{cfactor}}.
-#' @details Arguments passed via \code{...} to \code{cfactor} are only recycled 
-#'  if of length 1. Otherwise, an error is thrown. 
-#'  All arguments passed via \code{...} are applied in the order of the data 
+#' @details Arguments passed via \code{...} to \code{cfactor} are only recycled
+#'  if of length 1. Otherwise, an error is thrown.
+#'  All arguments passed via \code{...} are applied in the order of the data
 #'  columns but columns not to convert are skipped (see example).
-#' @return The original data frame is returned whereas the variables for which 
-#'  an encoding was provided are turned into (ordered) factors. All other 
+#' @return The original data frame is returned whereas the variables for which
+#'  an encoding was provided are turned into (ordered) factors. All other
 #'  columns are returned unmodified.
 #' @examples data <- data.frame(var1 = sample(x = 1:10, size = 20, replace = TRUE),
 #'                    var2 = rep(1:2, 20),
 #'                    var3 = sample(20),
-#'                    var4 = 2, 
+#'                    var4 = 2,
 #'                    var5 = sample(row.names(USArrests), size = 20),
 #'                    stringsAsFactors = FALSE)
-#' 
+#'
 #'  index <- data.frame(var = rep(paste0("var", 1:3), c(10, 2, 20)),
 #'                    encoding = c(1:10, 1:2, 1:20),
 #'                    label = c(letters[1:10], c("male", "female"), LETTERS[1:20]))
-#'                    
+#'
 #'  index_cfactor(data = data, index = index, variable = "var", ordered = c(TRUE, TRUE, FALSE))
 #' @export
 
-index_cfactor <- function(data, index, variable = "variable", 
-                          encoding = "encoding", label = "label", ...){
-  
-  
+index_cfactor <- function(data, index, variable = "variable",
+                          encoding = "encoding", label = "label", ...) {
   further_args <- list(...)
   # make sure variable, encoding and label are actual columns in the data frame
   pos_var <- match(variable, names(index))
   pos_enc <- match(encoding, names(index))
   pos_lab <- match(label, names(index))
-  check <- list(variable = pos_var, 
-                encoding = pos_enc, 
+  check <- list(variable = pos_var,
+                encoding = pos_enc,
                 label = pos_lab)
-  lapply(names(check), function(g) 
-    if(is.na(check[[g]])) {
-      stop(paste("argument '", g, "' specified incorrectly. ",
-                 "There is no such column in '", quote(index), "'", sep = ""))
-    }) 
-      
+  lapply(names(check), function(g)
+         if (is.na(check[[g]])) {
+           stop(paste("argument '", g, "' specified incorrectly. ",
+                      "There is no such column in '", quote(index), "'", sep = ""))
+  })
```

Comment:
Check R/indent-token-dependent.R 

```r
if (line_break_after(flattened_pd, target_index)) {

```
Would be more reasonable to check whether a comma follows on the same line instead of a line break right after '('?